### ISSUE TITLE
Incorrect question metadata after Update

### ISSUE THREAD

**Issue by tomeli5n** [metabase/metabase#40185]
### Describe the bug

![image](https://github.com/metabase/metabase/assets/7417323/4a35a65c-ea0d-4434-ae07-ea5843099369)

Version: 0.48.2

Description:

After updating to version 0.48.2, I've noticed an issue with the history of my questions. It appears that some questions have incorrect creation dates, showing the update date instead.

![image](https://github.com/metabase/metabase/assets/7417323/d7f93569-c1c8-4644-8660-a1c7ea0b7a88)

Additional context: The affected user account had been disabled years ago.

![image](https://github.com/metabase/metabase/assets/7417323/2e5a0602-7646-4aca-9780-2cf0d1971e02)

Other questions are experiencing the same issue.

![image](https://github.com/metabase/metabase/assets/7417323/1c713b7c-1ec8-46fd-94ad-e7785396ee3c)

### To Reproduce

1. Update to 0.48.2
2. Check on metadata questions.

### Information about your Metabase installation

```JSON
0.48.2 with MariaDB
```

### Severity

annoying only

**Comment by paoliniluis** [comment]
Can you move to 48.8?

**Comment by tomeli5n** [comment]
We continue to experience the error after updating to 48.8.

Analyzing the issue with the team, we see that it's not with all the questions, but with all those that have not been modified in the last few months and therefore do not have a change history. It seems that the bug created extra modification movements. For example, this question was definitely not modified.

![image](https://github.com/metabase/metabase/assets/7417323/7493d7df-8cee-486b-8635-041d1d366652)

**Comment by luizarakaki** [comment]
Is this still happening on 53? As this wasn't reported, I'm closing as being outdated

**Comment by tomeli5n** [comment]
@luizarakaki on 53
